### ISSUE TITLE
Greyscale styles

### ISSUE THREAD

**Issue by piperhaywood** [GemCopeland/personal-website#29]
@GemCopeland We should chat a little more about this. I think we should do it, but we might need to set a cookie since ideally, the user’s setting should be “remembered” if they continue to browse the site. Is that cool? You’ll probably need to include it in your “details” section.

**Comment by piperhaywood** [comment]
As discussed, I’ll implement this with cookie. I’ll pass it to @GemCopeland afterward to just add one short line about a cookie being set if greyscale is enabled.

**Comment by GemCopeland** [comment]
Here's the copy about the contrast settings:
_Your data isn’t collected when using this site. If you switch to high contrast mode, it will store a cookie to remember your preferences._

Do you think it makes sense?

Also here's the Unicode symbol for the switcher:
◑

**Comment by piperhaywood** [comment]
@GemCopeland I think that makes total sense. I tweaked it just a little to let people know how long it lasts for.

This is now implemented on `dev` btw

**Comment by GemCopeland** [comment]
Love it!
